### PR TITLE
Feature/download planet on container start

### PR DESCRIPTION
**Pull request by das-g**
#### Reviewed by:
- [x] @wasabideveloper 
- [x] @hixi
- [ ] @sfkeller

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    wget
+
+WORKDIR ${osm_planet_script_dir}
+
+ADD src ${osm_planet_script_dir}/src
+
+CMD ["/home/osm-planet/src/download_planet.sh"]
```
Comment (wasabideveloper) [inline Dockerfile:16]: Shouldn't this be 

``` python
CMD ["${osm_planet_script_dir}/src/download_planet.sh"]
```

?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+The `-ti` in there allows you to abort the download with **Ctrl-c**.
+The `--rm` removes the container when finished. (There's no good reason to keep it.)
+
+### advanced usage
+Pass the following environment variables to `docker run` with `--env <key>=<value>` to override the defaults:
+* **`osm_planet_mirror`** &mdash; OpenStreetMap planet mirror. See [OSM wiki](http://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors) for available servers.
+* **`osm_planet_path_relative_to_mirror`** &mdash; the file to download from the mirror. Defaults to `pbf/planet-latest.osm.pbf`
+* **`osm_planet_dir`** &mdash; where to place the downloaded file inside the container. Defaults to `/var/data/osm-planet`. You'll want to mount a host direcotry or another container's volume to the location you specify here, so that you can retrieve the downloaded file when the osm-planet container has been removed after running.
+
+Do **not** override `osm_planet_script_dir`.
```
Comment (wasabideveloper) [inline README.md:28]: Why? please add the reason.

File: Dockerfile (hunk 1 above)
Comment (das-g) [inline Dockerfile:16]: I agree that it **should**. Unfortunately, it **can't**: Environment variable expansion (or 'replacement' as the [documentation](http://docs.docker.com/engine/reference/builder/#environment-replacement) calls it) is only available in Dockerfile instructions `ADD`, `COPY`, `ENV`, `EXPOSE`, `LABEL`, `USER`, `WORKDIR`, `VOLUME`, `STOPSIGNAL`.

Comment (wasabideveloper) [inline Dockerfile:16]: :whale: :unamused:

File: README.md (hunk 2 above)
Comment (das-g) [inline README.md:28]: Now that I think about it: Overriding `osm_planet_script_dir` for `run` wouldn't have any effect, as the variable isn't used during runtime. (And docker doesn't allow setting environment variables from the command line for `build`.) But this variable was rather useless anyway, so I've removed it in 4b2e7b6 (PR #2).

Comment (wasabideveloper) [inline README.md:28]: :heavy_check_mark:

**Pull request merged**